services/meta/ota.md

```diff
@@ -1,88 +1,114 @@
 # OTA Service
-This service is used to manage over-the-air (OTA) upgrades of end devices.
+
+This service is used to manage over-the-air (OTA) updates of end devices.
 
 ## Service name
 
 `ota`
 
 ## Interfaces
 
-Type | Interface                   | Value type | Description                                                                                                      |
------|-----------------------------|------------|------------------------------------------------------------------------------------------------------------------|
-out  | evt.ota_progress.report     | int_map    | Shows update transfer progress and estimated time left, see [`ota_progress`](#definitions).                      |
-out  | evt.ota_start.report        | null       | Sent on upgrade start.                                                                                           |
-out  | evt.ota_end.report          | object     | Sent on upgrade end with upgrade status, see [`ota_end`](#definitions).                                          |
-in   | cmd.ota_status.get_report   | null       | Requests update progress information.                                                                            |
-out  | evt.ota_status.report       | object     | Shows update phase, see [`ota_status`](#definitions).                                                            |
-in   | cmd.ota_update.install      | string     | Z-Wave only. Starts an update of the device. Value should be the full path to the update                         | 
+| Type | Interface               | Value type | Description                                                                                                |
+|------|-------------------------|------------|------------------------------------------------------------------------------------------------------------|
+| in   | cmd.ota_update.start    | string     | Starts an update of the device. Value should be the full path to the firmware image file.                  |
```

Comment:
path could be in url format file:/ for local file or http:// if we want to update from the cloud.

https://en.wikipedia.org/wiki/URL

Reply:
This is something we can add in later when this will be implemented in the app etc.